### ISSUE TITLE
Timestamps are stored in UTC but not marked as such in the UI, and timezone is ignored when they’re written

### ISSUE THREAD

**Issue by strogonoff** [glossarist/glossarist-desktop#50]
Need to confirm that timestamps are stored correct in UTC format or with timezone, and fix if not (which apparently is the case)

**Comment by ronaldtse** [comment]
If there is a choice — only UTC please.

**Comment by strogonoff** [comment]
Storing UTC makes perfect sense. (The problem to be investigated is that during serialization _local time_ may be stored under the guise of UTC.)

**Comment by ronaldtse** [comment]
Yup, these implementation issues need to be tested and ironed out. Thanks!

**Comment by strogonoff** [comment]
I believe this is a Coulomb-level issue to sort out, in fact.

**Comment by ronaldtse** [comment]
@strogonoff okay, see if you need @phuonghuynh 's help anywhere.

**Comment by strogonoff** [comment]
I’ll deal with this Coulomb change but assigned to Phuong other issues regarding Glossarist specifically.
